styleguide/source/assets/js/main-navigation.js

```diff
@@ -1,36 +1,87 @@
 (function ($, Drupal) {
+
   Drupal.behaviors.ama_mainNavigation = {
     attach: function (context, settings) {
 
+      var $categoryNavWrapper = $('.ama_category_navigation_wrapper');
       var $categoryNavigationMenu = $('.ama_category_navigation_menu');
       var $mobileSearchTrigger = $('.global-search-trigger');
       var $mobileSearch = $('.ama__global-search');
+      var $viewportHeight = $(window).innerHeight();
+      var $mainNav = $('.ama__main-navigation ');
+      var $categoryNavMenuHeight = $('.ama_category_navigation_menu').outerHeight();
+      var $categoryNavMenuMResizedHeight = 0;
+
+      // Calculate whether or not the category nav should have scrollbars
+      function categoryNavHeight($resizeViewportHeight) {
```

Comment:
All variables in this function, save `$categoryNavWrapper`, hold integers. They should not have `$` in front of their names. Please update.